### PR TITLE
Remove 'end of file' comments

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: lib/quotsys.gd [hunk 1]
```diff
@@ -198,11 +198,6 @@ DeclareOperation( "GroupByQuotientSystem", [IsQuotientSystem] );
 #O  TraceDefinition . . . . . . trace a generator back to defining generators
 ##
 DeclareOperation( "TraceDefinition", [IsQuotientSystem, IsPosInt] );
-
```
Comment (alex-konovalov) [inline lib/quotsys.gd]: This is not an end-of-file comment

File: lib/pquot.gi [hunk 2]
```diff
@@ -1929,11 +1929,6 @@ InstallMethod( TraceDefinition,
     Add( trace, g );
     return Reversed( trace );
 end );
-
```
Comment (alex-konovalov) [inline lib/pquot.gi]: This is not an EOF comment.

File: lib/quotsys.gd (hunk 1 above)
Comment (fingolfin) [reply lib/quotsys.gd]: Oops, thanks!

**Comment by alex-konovalov** [comment]
New copyright headers and removing EOF comments is perhaps something that some package authors may be willing to adopt too.

**Comment by ChrisJefferson** [comment]
Before merging this, and the tab-to-space other commit, I thought we had previously agreed not to make large-scale formatting changes which didn't provide any significant gain, to avoid breaking people's forks (I have one, I know other people have them too).

**Comment by fingolfin** [comment]
Personally, I don't consider this PR to be a formatting change; I don't quite see how it could conflict with any other change either, unless that change for somevreason was modifying one of those end markers - or am I overlooking something?

As to the tab changes, I would prefer to discuss that on the relevant PR.

**Comment by alex-konovalov** [comment]
A merge conflict may happen when someone added next code at the end of the file, just above the end-of-file comment:
```
<<<<<<< HEAD

Let us put more text before the end of the file.

#####################
#End of file comments
##


=======
>>>>>>> <commit-id>
```
This is not hard to resolve, and it will only happen if the new code is appended at the end, though.

**Comment by ChrisJefferson** [comment]
Ok, j just aren't sure I feel the cost/benefit of this PR to be worth it, but then random comments on ends of files and similar things don't bother me at all. Others may have a different opinion.

**Pull request merged**